src/utils/docx/bl/exportLocation.js

```diff
@@ -0,0 +1,87 @@
+import {
+  Paragraph,
+  HeadingLevel,
+  Document,
+  Packer,
+  TextRun,
+  ExternalHyperlink,
+} from 'docx';
+import { saveAs } from 'file-saver';
+import {
+  writeLine,
+  style,
+  verticalSpace,
+  pageProperties,
+} from '../exportUtils';
+import writeLocationTable from './writeLocationTable';
+
+const getTitle = (title, latin) =>
```

Comment:
Looks like this can be moved to a generic util function and also be used by `lu`, right?

Reply:
Yes, done